releases/patch-releases.md

```diff
@@ -57,26 +57,29 @@ releases may also occur in between these.
 
 ### 1.19
 
-Next patch release is **1.19.4**
+Next patch release is **1.19.5**
 
 End of Life for **1.19** is **2021-09-30**
 
 | PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
 |--- |--- |--- |
+| 1.19.5        | 2020-12-11           | 2020-12-1   |
```

Comment:
We also cut 1.20.0-rc.0 at that date. I would prefer moving it to another date. How about December 9th?

Reply:
sure!